pychromecast/__init__.py

```diff
@@ -184,13 +169,7 @@ def get_chromecasts(
                      and returns a function which can be executed to stop discovery.
     :param callback: Callback which is triggerd for each discovered chromecast when
                      blocking = False.
-    :param blocking_app_launch: If set to true, start_app() will wait for the app
-                                to launch before returning. The default is to set
-                                blocking_app_launch = blocking.
     """
-    if blocking_app_launch is None:
-        blocking_app_launch = blocking
-
     if blocking:
```

Comment:
Still a blocking here?

Reply:
Right, this is a different `blocking`, I think it makes sense to keep it:
```
:param blocking: If True, returns a list of discovered chromecast devices.
                 If False, triggers a callback for each discovered chromecast,
                 and returns a function which can be executed to stop discovery.
```